zio-http/src/main/scala/zio/http/Method.scala

```diff
@@ -56,4 +66,6 @@ object Method {
   object DELETE  extends Method { val name = "DELETE"  }
   object TRACE   extends Method { val name = "TRACE"   }
   object CONNECT extends Method { val name = "CONNECT" }
+
+  object Default extends Method { val name = "GET" }
```

Comment:
Why implement this with another `object` and not just with a `def default = GET`? 🤔